### PR TITLE
D8CORE-2950: adding mailto icon to wysiwyg

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR advice

# Summary
This adds the mailto icon to the wysiwyg

# Needed By (Date)

# Urgency
- How critical is this PR?
not

# Steps to Test

1. pull this branch
1. Clear the cache
1. Create mailto links, or copy from https://amptesting.sites.stanford.edu/interior-page/wysiwyg/icons-links
1. In the editor you should see the email icons

Optionally if images aren't showing:
1. navigate to /stanford_profile_styles/node_modules
1. Delete the decanter directory (rm -rf decanter)
1. Clone decanter into /node_modules
1. Checkout this branch of Decanter: https://github.com/SU-SWS/decanter/pull/776
1. Build Decanter (npm run build)
1. Build this (npm run publish) to bring in the new images.

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket
https://stanfordits.atlassian.net/browse/D8CORE-2950
https://stanfordits.atlassian.net/browse/D8CORE-3169
- Other PRs
https://github.com/SU-SWS/decanter/pull/816
https://github.com/SU-SWS/decanter/pull/776
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
See issue:
https://github.com/SU-SWS/decanter/issues/621
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss [hunk 1]
```diff
@@ -7,4 +7,20 @@
 // acceptable amount of bloat for maintaining styles automatically.
 body {
   @extend .su-wysiwyg-text;
+
+  a[href^="mailto:"] {
+    &::after {
+      @include padding(null null null 0.1rem);
+      content: "\f0e0";
+      color: $su-color-bright-blue;
+      font-family: "Font Awesome 5 Free";
```
Comment (pookmish) [inline modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss:16]: I don't believe Font Awesome styles are added in the wysiwyg. We can probably get away with just a simple png for the wysiwyg.

**Review by pookmish — CHANGES_REQUESTED** [review]
This functionality doesn't seem to work at the moment. I see a space after a mail to link, but no icon. 
image

File: modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss (hunk 1 above)
Comment (cjwest) [reply modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss:16]: Ahhh! Thank you! I'll give it a go

File: modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss [hunk 2]
```diff
@@ -7,4 +7,26 @@
 // acceptable amount of bloat for maintaining styles automatically.
 body {
   @extend .su-wysiwyg-text;
+
+  a[href^="mailto:"] {
+    &::after {
+      @include margin(null 0.1em null 0.25em);
+      @include padding(null 0.01em null null);
+      background-image: url("#{$su-image-path}/arrow-down.png");
```
Comment (pookmish) [inline modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss:15]: Something isn't compiling correctly with this path. 
image

**Review by pookmish — CHANGES_REQUESTED** [review]

**Comment by pookmish** [comment]
Unassigned myself. feel free to reassign when the issues are ready for review

**Comment by cjwest** [comment]
@rebeccahongsf,
I would love your help in building proper .pngs for all these mailto cases.

**Comment by cjwest** [comment]
From Inside the WYSIWYG:
Screen Shot 2021-03-31 at 6 08 30 PM
Screen Shot 2021-03-31 at 6 08 16 PM
Screen Shot 2021-03-31 at 6 07 55 PM
Screen Shot 2021-03-31 at 6 07 17 PM

**Comment by cjwest** [comment]
See PR: https://github.com/SU-SWS/decanter/pull/816

**Comment by pookmish** [comment]
This hasn't moved in almost a year. I'm closing the PR

**Pull request closed without merging**